libqtile/resources/default_config.py

```diff
@@ -52,7 +52,7 @@
     # Unsplit = 1 window displayed, like Max layout, but still with
     # multiple stack panes
     Key([mod, "shift"], "Return", lazy.layout.toggle_split()),
-    Key([mod], "Return", lazy.spawn("xterm")),
+    Key([mod], "Return", lazy.spawn("qtile-terminal")),
```

Comment:
Can we change this to a python function that looks something like lazy.spawn_reasonable_terminal()? It would be great not to have to distribute another "binary" with everything.

Reply:
the first idea was as you suggest. Thou I wanted to give a try. But I agree with you. Other idea is to set default terminal and use `lazy.spawn_terminal()` or ex. `lazy.spawn_terminal('tmux')`, so the command could be executed. Then in config we can add `default_terminal = None` as default and ex. `default_terminal = 'st'` would run in `st`

Reply:
Really like this idea, over the binary. 
I imagine, most people already have their pick for default terminal and this would support that.